.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "editor.formatOnSave": true,
+  "editor.tabSize": 2,
+  "editor.codeActionsOnSave": {
+    "source.fixAll": true,
+    "prettier.singleQuote": true,
```

Comment:
as a result this line is obsolete 
https://github.com/Amoodaa/react-context-workshop/blob/master/.vscode/settings.json

Reply:
thanks, a lot Emad , we are stack on this from the morning.